spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/filter/ratelimit/RedisRateLimiterTests.java

```diff
@@ -47,11 +48,12 @@ public void redisRateLimiterWorks() throws Exception {
 		int replenishRate = 10;
 		int burstCapacity = 2 * replenishRate;
 
-		String routeId = "myroute";
-		rateLimiter.getConfig().put(routeId, new RedisRateLimiter.Config()
+		rateLimiter.getConfig().put(RouteDefinitionRouteLocator.DEFAULT_FILTERS+1, new RedisRateLimiter.Config()
```

Comment:
There needs to be a new test so that this old test (configuring with the routeId) continues to be tested to avoid regression.